front50-web/src/test/groovy/com/netflix/spinnaker/front50/controllers/NotificationControllerTck.groovy

```diff
@@ -188,15 +189,17 @@ class S3NotificationControllerTck extends NotificationControllerTck {
   def scheduler = Schedulers.from(Executors.newFixedThreadPool(1))
 
   @Shared
-  S3NotificationDAO s3NotificationDAO
+  NotificationDAO notificationDAO
 
   @Override
   NotificationDAO createNotificationDAO() {
     def amazonS3 = new AmazonS3Client(new ClientConfiguration())
     amazonS3.setEndpoint("http://127.0.0.1:9999")
     S3TestHelper.setupBucket(amazonS3, "front50")
 
-    s3NotificationDAO = new S3NotificationDAO(new ObjectMapper(), amazonS3, scheduler, 0, "front50", "test")
-    return s3NotificationDAO
+    def storageService = new S3StorageService(new ObjectMapper(), amazonS3, "front50", "test")
```

Comment:
These were some previous tests that could be pointed at an S3 proxy running locally on a laptop.

Honestly rarely ran beyond when S3 support was originally added.